include/flatbuffers/flatbuffer_builder.h

```diff
@@ -782,7 +782,7 @@ class FlatBufferBuilder {
   /// where the vector is stored.
   template<typename T>
   Offset<Vector<const T *>> CreateVectorOfStructs(const T *v, size_t len) {
-    StartVector(len * sizeof(T) / AlignOf<T>(), AlignOf<T>());
+    StartVector(len * sizeof(T) / AlignOf<T>(), sizeof(T), AlignOf<T>());
```

Comment:
So is this the change that is required to fix the test case?

Reply:
Yes, this is the important part to fix the test case